src/main/java/picard/analysis/directed/RnaSeqMetricsCollector.java

```diff
@@ -430,20 +430,34 @@ else if (this.metrics.SAMPLE != null) {
             return out;
         }
 
+        protected Set<Gene> getGenesForPickTranscripts() {
```

Comment:
Is the idea that a subclass would want to change this method, or would need access to it? In either case it would be helpful to add a javadoc about what it does (conceptually) in this class. If it can be made private, then it would be OK IMO to not bother adding a javadoc.

Reply:
Yes, it is there for a subclass to override.  I added javadoc.